src/site/layouts/health_care_local_facility_page.drupal.liquid

```diff
@@ -165,6 +165,49 @@
+                    {% if fieldFacebook != empty || fieldFlickr != empty || fieldInstagram != empty || fieldTwitter != empty || fieldEmailSubscription != empty %}
+                        <section class="feature feature-grey">
+                            <h3>Get updates from us</h3>
+                            <div class="va-c-list-link-teasers">
+                                <div class="usa-grid usa-grid-full">
+                                    {% if fieldFacebook != empty %}
+                                        <div class="social-links social-links-facebook usa-width-one-half vads-u-margin-bottom--2">
+                                            <a target="_blank" href="{{ fieldFacebook.url.path }}"><i class="fab fa-facebook"></i> {{ fieldFacebook.title }}</a>
+                                        </div>
+                                    {% endif %}
+
+                                    {% if fieldTwitter != empty %}
+                                        <div class="social-links social-links-twitter usa-width-one-half vads-u-margin-bottom--2">
+                                            <a target="_blank" href="{{ fieldTwitter.url.path }}"><i class="fab fa-twitter"></i> {{ fieldTwitter.title }}</a>
+                                        </div>
+                                    {% endif %}
+                                </div>
+
+                                <div class="usa-grid usa-grid-full">
+                                    {% if fieldFlickr != empty %}
+                                        <div class="social-links social-links-flickr usa-width-one-half vads-u-margin-bottom--2">
+                                            <a target="_blank" href="{{ fieldFlickr.url.path }}"><i class="fab fa-flickr"></i> {{ fieldFlickr.title }}</a>
+                                        </div>
+                                    {% endif %}
+
+                                    {% if fieldInstagram != empty %}
+                                        <div class="social-links social-links-instagram usa-width-one-half vads-u-margin-bottom--2">
+                                            <a target="_blank" href="{{ fieldInstagram.url.path }}"><i class="fab fa-instagram"></i> {{ fieldInstagram.title }}</a>
+                                        </div>
+                                    {% endif %}
+                                </div>
+
+                                <div class="usa-grid usa-grid-full">
+                                    {% if fieldEmailSubscription != empty %}
+                                        <div class="social-links social-links-email usa-width-one-half vads-u-margin-bottom--2">
+                                            <a target="_blank" href="{{ fieldEmailSubscription.url.path }}"><i class="fas fa-envelope"></i> {{ fieldEmailSubscription.title }}</a>
+                                        </div>
+                                    {% endif %}
```

Comment:
Should the if-statement wrap the `<div class="usa-grid usa-grid-full">` as well?

Reply:
Yeah, you are right - I was just being lazy, @ncksllvn